qa/distros/podman/centos_8.2_container_tools_3.0.yaml

```diff
@@ -1,5 +1,4 @@
 os_type: centos
-os_version: "8.2"
```

Comment:
we were planning to get rid of this in https://github.com/ceph/ceph/pull/44868/commits/8f1d1b7c55a6a5715e12e9586be0951fad15cf49, @adk3798 thoughts?